sources/ToolJobs.m

```diff
@@ -93,6 +93,20 @@ - (void)setIntValue:(int)i {
     [self setToolTip:[NSString stringWithFormat:@"SIG%@ (%d)", signalNames[i], i]];
 }
 
+- (NSUInteger)uintValue {
```

Comment:
No need to change `ToolJobs.m`. The only caller of `setIntValue:` in this class is `textDidEndEditing:` just below.